frontend/src/views/Profile/Profile.tsx

```diff
@@ -18,10 +18,20 @@ const profileDisplayOptions = [
 
 async function getProfile(profileID: string): Promise<ProfileInterface | null> {
     try {
-        const profileInfo = await fetch(`http://localhost:8000/profiles/${profileID}`);
-        return profileInfo.json();
-    } catch (err) {
-        return null;
+        const response = await fetch(`http://localhost:8000/profiles/${profileID}`, {
```

Comment:
same here - the fetch should be moved out of the component, and to make it more reusable you can later use axios with the baseurl declared in .env